python_modules/dagstermill/dagstermill/__init__.py

```diff
@@ -390,10 +383,10 @@ def _t_fn(info, inputs):
                                 current_time = new_time
 
             if process.returncode != 0:
-                # Throw event that is an execution error!
-                info.log.debug("There was an error in Papermill!")
-                info.log.debug('stderr was None' if process.stderr is None else process.stderr)
-                exit()
+                raise DagstermillError(
+                    "There wsas an error when Papermill tried to execute the notebook. "
```

Comment:
nit: single quote plz

Reply:
Fixed in materialization_event pull request